configs/grafana/dashboards/telemetry-dashboard.json

```diff
@@ -6798,7 +6798,7 @@
           "query": "label_values(network_instance_name)",
           "refId": "StandardVariableQuery"
         },
-        "refresh": 1,
```

Comment:
what is this mode exactly? Don't remember seeing it

Reply:
This means that on every chnage of timerange the variable value range (for the dropdown var selection) is recalculated.
I've had some trouble figuring out that I would need to reload the whole dashboard, because I assessed is befor the data was populate. That lead to me search for an issue in the whole setup... so for such kind of lab env I think this is better suited.